src/main/java/com/faforever/client/main/event/NavigateEvent.java

```diff
@@ -1,7 +1,9 @@
 package com.faforever.client.main.event;
 
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
 
+@EqualsAndHashCode
 public class NavigateEvent {
   @Getter
   private final NavigationItem item;
```

Comment:
I'd suggest using `@Value` and `@AllArgsConstructor`.
Then remove `@EqualsAndHashCode`, remove `@Getter private final` and remove the constructor